test/image/mocks/bar_stackrelative_negative.json

```diff
@@ -0,0 +1,13 @@
+{
+    "data":[
+        {"name":"Col1","y":["-1","2","3","4","5"],"x":["1","2","3","4","5"],"type":"bar","uid":"aeb9ea"},
```

Comment:
🐄  Fix the alignment to be typical prettyprinted json. Single-line objects are fine if they fit in < 80 characters, but otherwise break them up.